common/src/View/MoveBrushVerticesCommand.cpp

```diff
@@ -71,6 +75,10 @@ namespace TrenchBroom {
             return true;
         }
 
+        std::unique_ptr<CommandResult> MoveBrushVerticesCommand::doCreateCommandResult(const bool success) {
+            return std::make_unique<MoveBrushVerticesCommandResult>(success, !m_newVertexPositions.empty());
+        }
+
         bool MoveBrushVerticesCommand::doCollateWith(UndoableCommand* command) {
             MoveBrushVerticesCommand* other = static_cast<MoveBrushVerticesCommand*>(command);
```

Comment:
dynamic_cast

Reply:
Here, the command type is checked by the superclass, so this cast is perfectly safe. I'm generally careful to static_cast only when I'm sure the cast cannot go wrong. I'm not going to change it here as such code is all over the commands. I'd rather we put this in a separate maintenance issue, where we also should put this into our coding standards document.